TabPanel: Add Overview (#9078)
DevExpress/devextreme-documentation: concepts/05 UI Components/TabPanel/02 Overview/00 Overview.md

> DevExtreme TabPanel is a UI component that displays content in panes and allows users to switch between items using tabs.

Reviewer: ```suggestion
DevExtreme TabPanel is a UI component that displays content in panes and allows users to switch between them using tabs.
```

> In this overview you can find a map of TabPanel elements, information about key features, and what to explore next.

Reviewer: ```suggestion
This overview includes a map of TabPanel elements, information about key features, and suggested next steps.
```
